modules/nf-core/fq/subsample/meta.yml

```diff
@@ -0,0 +1,42 @@
+name: "fq_subsample"
+description: fq subsample outputs a subset of records from single or paired FASTQ files.
```

Comment:
The function requires a random seed. It would be good to make it explicit that this needs to be set.